src/routes/docs/tooling/command-line/buckets/+page.markdoc

```diff
@@ -0,0 +1,119 @@
+---
+layout: article
+title: Deploy buckets with CLI
+description: Efficiently deploy your Appwrite buckets using the Command-Line Tool (CLI). 
+---
+
+{% info title="Before proceeding" %}
+Ensure you [**install**](/docs/tooling/command-line/installation#Installation) the CLI, [**login**](/docs/tooling/command-line/installation#get-started) to your Appwrite account, and [**initialize**](/docs/tooling/command-line/installation#Initialization) your Appwrite project.
+{% /info %}
+
+The Appwrite CLI allows you to configure and deploy buckets across projects.
+
+# Initialize bucket {% #initialize-bucket %}
+
+Initialize your new bucket using the following command:
+
+```sh
+appwrite init bucket
+```
+
+# Pull bucket {% #pull-bucket %}
+
+You can also pull your existing Appwrite buckets using the `pull` command in the folder holding your `appwrite.json` file.
```

Comment:
It's not very common to refer to a folder holding something. I think it's more common to see "containing"